tests/test_pipelines.py

```diff
@@ -320,8 +238,9 @@ def test_tf_fill_mask(self):
                 },
             ],
         ]
-        for tokenizer, model, config in TF_FILL_MASK_FINETUNED_MODELS:
-            nlp = pipeline(task="fill-mask", model=model, config=config, tokenizer=tokenizer, framework="tf", topk=2)
+        for model_name in TF_FILL_MASK_FINETUNED_MODELS:
+            tok_arg = (model_name, {"use_fast": False})
```

Comment:
I don't think we need the ` {"use_fast": False}` (especially as we don't actually test outputs anymore)